proto/swap_resolver.proto

```diff
@@ -0,0 +1,106 @@
+
+syntax = "proto3";
+
+//import "google/api/annotations.proto";
+
+//option java_multiple_files = true;
+//option java_package = "io.grpc.examples.routeguide";
+//option java_outer_classname = "RouteGuideProto";
+
+package swapresolver;
+
+// Interface exported by the server.
+service SwapResolver {
+  // ResolveHash is used by LND to request translation of Rhash to a pre-image.
+  // the resolver may return the preimage and error indicating that there is no
+  // such hash/deal
+  rpc ResolveHash(ResolveReq) returns (ResolveResp) {}
+
+}
+
+enum CoinType {
+    BTC = 0;
```

Comment:
Mainly a note to self here, but eventually we'll want to replace this because we don't want to be hardcoding supported currencies long term.

Reply:
Yes. But this is LND part so we will see about it